src/cli/commands/drives.rs

```diff
@@ -72,15 +70,11 @@ pub enum DrivesCommand {
 
 #[async_trait(?Send)]
 impl RunnableCommand<NativeError> for DrivesCommand {
-    async fn run_internal(
-        self,
-        global: &mut GlobalConfig,
-        client: &mut Client,
-    ) -> Result<String, NativeError> {
+    async fn run_internal(self) -> Result<String, NativeError> {
         match self {
             // List all Buckets tracked remotely and locally
             DrivesCommand::Ls => {
-                let omnis = OmniBucket::ls(global, client).await;
```

Comment:
whats up with this interface change

Reply:
what im getting at is i asked to be unblocked on login, this change couldve been encapulsated in another ticket . change. its fine but just unexepcted